letsencrypt/client/init.sls

```diff
@@ -13,6 +13,7 @@ certbot_packages:
     - names: {{ client.source.pkgs }}
     - watch_in:
       - cmd: certbot_installed
+    - fromrepo: jessie-backports
```

Comment:
After configuring prefereces.d as advised, now I get *bunch of* packages wanting to upgrade from backports. I do not want all my packages being installed from backports, only specific ones! Please reconsider pillar suggestion, by implementing "pkg_fromrepo" optional key or similar.